src/pages/Welcome/Welcome.js

```diff
@@ -1,30 +1,46 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { Button } from '@material-ui/core';
-import getToken from '../../lib/tokens';
-import { db } from '../../lib/firebase';
+import CreateOrJoin from '../../components/CreateOrJoin';
 import ShareToken from '../../components/ShareToken';
 
-const Welcome = (props) => {
-  const storeToken = () => {
-    const token = getToken();
-    props.updateToken(token);
-    db.collection(token).doc('ListData').set({ listCreated: new Date() });
-  };
-  return (
-    <div>
-      <h2>Welcome to your Smart Shopping List!</h2>
-      <Button
-        color="primary"
-        onClick={() => storeToken()}
-        type="button"
-        variant="contained"
-      >
-        Create a new list
-      </Button>
-      <hr />
-      <ShareToken updateToken={props.updateToken} />
-    </div>
-  );
-};
+// const Welcome = (props) => {
```

Comment:
Oh, I'm curious why you updated this to a Class...

Reply:
@laurenmbeatty @luisaugusto This is the way I learned to make components stateful - I know there's an updated way to include state in functional components, but I'm less familiar with it and did it this way to avoid errors. If there's a cleaner way to achieve this, I'm happy to make the changes!

Reply:
No worries! In the future, you can use the `useState` hook in functional to include a state. You can see an example in the [React Docs](https://reactjs.org/docs/hooks-state.html)